src/Sylius/Bundle/CoreBundle/Resources/config/doctrine/model/ChannelPricing.orm.xml

```diff
@@ -27,7 +27,7 @@
         </id>
         <field name="price" column="price" type="integer" nullable="true" />
         <field name="originalPrice" column="original_price" type="integer" nullable="true" />
-        <field name="minimumPrice" column="minimum_price" type="integer" nullable="true" />
+        <field name="minimumPrice" column="minimum_price" type="integer" nullable="false" />
```

Comment:
We should probably define the default value as 0 - it would also prevent breaking the already existing data